docs/client/rest/rest_api.md

```diff
@@ -371,6 +373,23 @@ Create a new batch.
 
 The created [Batch](#batch) object.
 
+### POST /batches (with uploading resource)
+
+Create a new batch with uploading resource file.
+
+#### Request Body
+- Media type: `multipart-formdata`
+- Request body structure in multiparts:
+
+| Name         | Description                                                                                      | Media Type       |
+|:-------------|:-------------------------------------------------------------------------------------------------|:-----------------|
+| batchRequest | The batch request in JSON format as request body requried in [POST /batches](#post-batches)      | application/json |
+| resourceFile | The uploading resource to execute, which will be cached on server and cleaned up after execution | File             |
```

Comment:
The resource to upload and execute?

Reply:
could you provide the `curl command` for this?

Reply:
Good idea and the curl example is added to the API description.